CHANGELOG

```diff
@@ -1,3 +1,13 @@
+3.14.0.dev0
```

Comment:
This is the only thing I don't understand in this PR, which is superb, btw.
This version is going to look higher than `3.14.0`. I would have expected `3.13.0.dev1` or `3.13.0~dev1` to make sure it falls between the old `3.13.0` and the upcoming `3.14.0`.

Reply:
As discussed, the version number follows PEP 440. I wanted to do a test using `pip` to show that it is sorted below `3.14.0`, but somehow if given a git repository, `pip` always installs the version found in the given (or master) branch, even if it is older than the installed one. I seem to remember that it did some version check for this and skipped installed or higher versions, but I am unable to reproduce it. Maybe `pip` only does version comparisons for packages on `pypi`?

This version scheme would not be correct for Debian packages, but I am anyhow planning to increase the version to `3.14.0` once this is merged into master.